### PR TITLE
(DOCSP-20326): January 19 2022 Release Notes

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
## Pull Request Info

- there is a broken ref in here, but it will be fixed by merged-in flexible sync docs before this goes out.

### Jira

- https://jira.mongodb.org/browse/DOCSP-20326

### Staged Changes (Requires MongoDB Corp SSO)

- [Release Notes](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-20326/release-notes/backend/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by MongoCaleb — APPROVED** [review]
lgtm

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.. _backend_20220114:
+
+14 January 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Refreshed the Sync Schema UI.
```
Comment (sumedhamehta) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:6]:
```suggestion
```

Let's remove this it will not be ready in time

**Review by sumedhamehta — CHANGES_REQUESTED** [review]

**Comment by nathan-contino-mongo** [comment]
@sumedhamehta is this out?

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.. _backend_20220114:
+
+14 January 2022 Release
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:3]:
```suggestion
19 January 2022 Release
```

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.. _backend_20220114:
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:1]:
```suggestion
.. _backend_20220119:
```

**Pull request merged**